core/tests/webdriverio_desktop/additionalEditorFeatures.js

```diff
@@ -66,8 +66,8 @@ describe('Full exploration editor', function() {
       await users.login('user@heightWarning.com');
 
       await workflow.createExploration(true);
-
-      var postTutorialPopover = $('.ng-joyride .popover-content');
+      // eslint-disable-next-line oppia/e2e-practices
```

Comment:
Why is this added?

Reply:
![Screenshot from 2022-10-27 09-44-31](https://user-images.githubusercontent.com/77006897/198190120-286134a5-4d8d-494e-92c8-ad790ded70bd.png)

Reply:
removed